src/@types/index.ts

```diff
@@ -39,6 +39,26 @@ export interface PluginStringMap {
     replace: strings;
 }
 
+export type RTLCSSPlugin = {
+    name: string;
+    priority: number;
+    directives: {
+        control: Object,
+        value: Array<Object>
```

Comment:
The linting job is failing, just change this `Objects` to `object` and it should fix the issue.